### ISSUE TITLE
✅ 🐞 Bug Report: Border control does not work as expected for nested Accordions inside nested Accordions - [ED-11616]

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#23058]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi team,

When I place an accordion widget inside another accordion, the border control does not work as expected. It seems that this bug only occurs for border control in Style tab > accordion section.

### Steps to reproduce

https://github.com/elementor/elementor/assets/77163248/11c6de34-e807-49ab-be6c-f8c6625c3020

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt



== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.38 (Win32) OpenSSL/1.1.1a PHP/7.3.2
	MySQL version: mariadb.org binary distribution v5.6.42-84.2
	PHP Version: 7.3.2
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: http://localhost:85/elementor-beta
	Home URL: http://localhost:85/elementor-beta
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.15.0-beta2
		Author: Elementor.com


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Hello Theme Header &amp; Footer: Inactive
	Editor Top Bar: Active
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Global Style Guide: Active


== Log ==
	
Log: showing 20 of 342023-06-08 20:36:09 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta1',
  'to' => '3.14.0-beta3',
)]
2023-06-08 20:36:10 [info] elementor::elementor_updater Started 
2023-06-08 20:36:10 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-08 20:36:10 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-08 20:36:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta1',
  'to' => '3.14.0-beta3',
)]
2023-06-12 17:44:23 [info] elementor::elementor_updater Started 
2023-06-12 17:44:23 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-12 17:44:23 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-12 17:44:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta3',
  'to' => '3.14.0-beta4',
)]
2023-06-12 17:44:23 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta3',
  'to' => '3.14.0-beta4',
)]
2023-06-14 11:32:35 [info] elementor::elementor_updater Started 
2023-06-14 11:32:35 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-14 11:32:35 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-14 11:32:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta4',
  'to' => '3.14.0-beta5',
)]
2023-06-14 11:32:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta4',
  'to' => '3.14.0-beta5',
)]
2023-07-09 18:14:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta5',
  'to' => '3.15.0-beta2',
)]
2023-07-09 18:14:15 [info] elementor::elementor_updater Started 
2023-07-09 18:14:16 [info] Elementor/Upgrades - _on_each_version Start  
2023-07-09 18:14:16 [info] Elementor/Upgrades - _on_each_version Finished 
2023-07-09 18:14:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.0-beta5',
  'to' => '3.15.0-beta2',
)]

JS: showing 3 of 3JS: 2023-05-02 12:23:32 [error X 37][http://localhost:85/elementor-beta/wp-admin/load-scripts.php?c=1&amp;loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&amp;loadchunk_1=hooks&amp;ver=6.1.1:4:51824] Cannot read properties of undefined (reading &#039;replace&#039;) 
JS: 2023-06-02 18:44:11 [error X 1][http://localhost:85/elementor-beta/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.14.0-beta1:2:2932] elementor_new_template_form_controls is not defined 
JS: 2023-07-10 20:49:33 [error X 2][http://localhost:85/elementor-beta/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:7826] Cannot read properties of null (reading &#039;dataset&#039;) 

PHP: showing 5 of 5PHP: 2023-05-28 19:04:05 [notice X 3][C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\core\common\modules\ajax\module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-02 18:31:07 [notice X 1][C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\app\modules\import-export\module.php::734] Trying to get property &#039;label&#039; of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-12 18:03:20 [error X 2][C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\modules\ai\module.php::492] Uncaught Error: Cannot use object of type WP_Error as array in C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\modules\ai\module.php:492
Stack trace:
#0 C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\modules\ai\module.php(468): Elementor\Modules\Ai\Module-&gt;upload_image(&#039;https://storage...&#039;, &#039;The image of a ...&#039;, 2771)
#1 [internal function]: Elementor\Modules\Ai\Module-&gt;ajax_ai_upload_image(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\core\common\modules\ajax\module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 C:\xampp\htdocs\elementor-beta\wp-includes\class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request(&#039;&#039;)
#4 C:\xampp\htdocs\elementor-beta\wp-includes\class-wp-hook.php(332): WP_Hook-&gt;apply_filters(&#039;&#039;, Array)
#5 C:\xampp\htdocs\elementor-beta\wp-includes\plugin.php(517): WP_Hook-&gt;do_action(Array)
#6 C:\xampp\htdocs\elementor-bet [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-09 19:10:54 [notice X 3][C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\modules\ai\module.php::569] Undefined index: prompt [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-07-12 19:22:26 [error X 3][C:\xampp\htdocs\elementor-beta\wp-content\plugins\elementor\includes\settings\tools.php::9] Maximum execution time of 30 seconds exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	



````

</details>

**Comment by 79ho3ein** [comment]
Hi @nicholaszein, Please check this thread if possible and also [this one](https://github.com/elementor/elementor/issues/23062) which is related to beta 3.15.

**Comment by nicholaszein** [comment]
@79ho3ein,

Thanks for sharing your feedback.

It might not seem like I am, but I do visit all new issues and feature requests every day. We are very aware of the issues reported. It's just not time to triage then yet! 😉

**Comment by nicholaszein** [comment]
Hello there @79ho3ein!

### This issue was solved in Elementor Core v3.15 - Beta 4! 🥳 

Thank you for your contribution! 🙏 

Cheers 🥂

**Comment by nicholaszein** [comment]
Hi @79ho3ein,

I'm reopening this issue to continue our [conversation](https://github.com/orgs/elementor/discussions/23010#discussioncomment-6585179) from the [3.15-Beta](https://github.com/orgs/elementor/discussions/23010) discussion.

Just to facilitate the organization, here's a quote from your [follow-up comment](https://github.com/orgs/elementor/discussions/23010#discussioncomment-6585945):

> Hi @nicholaszein,
> 
> I don't think it is related to cache and other similar cases. From what I saw in your video, it seems that you checked this issue immediately after adding the nested accordion widget, and I don't see any bugs in this situation either (on my localhost). But when I make changes in the widget settings, I encounter this bug. Changes like Space between Items - border (active and hover).
> 
> Please export the template in Elementor and check the bug again. Is there still a problem?
> 
> [elementor-3270-2023-07-30.zip](https://github.com/elementor/elementor/files/12208633/elementor-3270-2023-07-30.zip)

Please allow me some time to investigate it, and get back to you with my findings. 🧐 

Kind regards

**Comment by nicholaszein** [comment]
@79ho3ein,

I found out what is the issue! 🙌 

I'll report to the team and see if we can fix it. 

Thank you 🙏

**Comment by nicholaszein** [comment]
Hey @79ho3ein!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
